rotkehlchen/db/updates.py

```diff
@@ -154,6 +162,47 @@ def update_rpc_nodes(self, to_version: int) -> None:
             new_default_nodes=new_default_nodes,
         )
 
+    def update_global_addressbook(self, from_version: int, to_version: int) -> None:
+        """Downloads and applies global addressbook updates from from_version to to_version"""
+        db_addressbook = DBAddressbook(self.user_db)
+        for version in range(from_version + 1, to_version + 1):
+            success, new_raw_entries = self._get_file_content(
+                file_url=GLOBAL_ADDRESSBOOK_URL.format(branch=self.branch, version=version),
+                update_type=UpdateType.GLOBAL_ADDRESSBOOK,
+            )
+            if success is False:
+                log.error(f'Failed to download globaldb addressbook for version {version}')
+                continue
+
+            entries_to_add = []
+            for raw_entry in new_raw_entries:
+                entry = AddressbookEntry(
+                    address=raw_entry['address'],
+                    name=raw_entry['name'],
+                    blockchain=SupportedBlockchain.deserialize(raw_entry['blockchain']) if raw_entry['blockchain'] else None,  # noqa: E501
+                )
+                existing_name = db_addressbook.get_addressbook_entry_name(
+                    book_type=AddressbookType.GLOBAL,
+                    chain_address=OptionalChainAddress(
+                        address=entry.address,
+                        blockchain=entry.blockchain,
+                    ),
+                )
+                if existing_name is None:
+                    entries_to_add.append(entry)
+                else:
+                    log.debug(
+                        f'Global address book entry for address {entry.address} and blockchain '
+                        f'{entry.blockchain} already exists (probably was input by the user). '
+                        f'Update for this entry will not be applied.',
+                    )
```

Comment:
In the other PR (curve api pr) Lefteris said that we should not add things like `exists_ok` flag to `add_addresbook_entries` method, so I did it like this here. I'll also add the check here whether to force insert remote name, and for this it will probably be needed to get current local names.